// @angular-material-components/file-input/esm2020/lib/file-input.module.mjs
import { CommonModule } from '@angular/common';
import { NgModule } from '@angular/core';
import { MatButtonModule } from '@angular/material/button';
import { MatFormFieldModule } from '@angular/material/form-field';
import { MatIconModule } from '@angular/material/icon';
import { MatInputModule } from '@angular/material/input';
import { NgxMatFileInputComponent, NgxMatFileInputIcon } from './file-input.component';
import * as i0 from "@angular/core";
export class NgxMatFileInputModule {
}
/** @nocollapse */ /** @nocollapse */ NgxMatFileInputModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.0.1", ngImport: i0, type: NgxMatFileInputModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
/** @nocollapse */ /** @nocollapse */ NgxMatFileInputModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.0.1", ngImport: i0, type: NgxMatFileInputModule, declarations: [NgxMatFileInputComponent,
        NgxMatFileInputIcon], imports: [CommonModule,
        MatButtonModule,
        MatIconModule,
        MatFormFieldModule,
        MatInputModule], exports: [NgxMatFileInputComponent,
        NgxMatFileInputIcon] });
/** @nocollapse */ /** @nocollapse */ NgxMatFileInputModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.0.1", ngImport: i0, type: NgxMatFileInputModule, imports: [[
            CommonModule,
            MatButtonModule,
            MatIconModule,
            MatFormFieldModule,
            MatInputModule
        ]] });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.0.1", ngImport: i0, type: NgxMatFileInputModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [
                        NgxMatFileInputComponent,
                        NgxMatFileInputIcon
                    ],
                    imports: [
                        CommonModule,
                        MatButtonModule,
                        MatIconModule,
                        MatFormFieldModule,
                        MatInputModule
                    ],
                    exports: [
                        NgxMatFileInputComponent,
                        NgxMatFileInputIcon
                    ]
                }]
        }] });
